sdk/keyvault/keyvault-keys/test/public/utils/crypto.ts

```diff
@@ -1,28 +1,14 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import { isNode } from "@azure/core-util";
-import { JsonWebKey } from "../../../src";
+import { JsonWebKey } from "../../../src/index.js";
 
 export function stringToUint8Array(str: string): Uint8Array {
-  if (isNode) {
-    return new Uint8Array(Buffer.from(str));
-  } else {
-    const bytes = new Uint8Array(str.length);
-    for (let i = 0; i < str.length; i++) {
-      bytes[i] = str.charCodeAt(i);
-    }
-    return bytes;
-  }
+  return new Uint8Array(Buffer.from(str));
 }
 
 export function uint8ArrayToString(ab: Uint8Array): string {
-  if (isNode) {
-    return Buffer.from(ab).toString("utf-8");
-  } else {
-    const decoder = new TextDecoder("utf-8");
-    return decoder.decode(ab);
-  }
+  return Buffer.from(ab).toString("utf-8");
```

Comment:
nitpick: perhaps we could use `TextDecoder` instead to future-proof it?

Reply:
I'll keep that in mind but not for this PR - it's a little hard to bundle changes into this at this point